### PR TITLE
Updated to work with Riot v3 + fixed some typos

### PR DESCRIPTION
**Pull request by jonrandy**
I think I've done everything necessary to make it work with Riot v3. Getting a fail on one test though, with the 'not found' event - struggling to work out why

### CODE REVIEW

File: tests/isomorphism.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ riot.tag2('router', '<div id="riotcontainer" class="route-container"><yield></yi
 
 			setTimeout(function () {
 				throw e;
-			}, 0);
+			}, 3000);
```
Comment (prateekbh) [inline tests/isomorphism.js]: why the delay?

**Comment by prateekbh** [comment]
Hey thanks for the work!
just left some comments for you

**Comment by jonrandy** [comment]
Ah - that was just a hangover from me testing something to try and figure out what was going on with the failing test. I've removed it

**Comment by prateekbh** [comment]
Awesome...
So sorry I dont know how but I guess travis integration was removed, can you verify if npm run tests is not failing

**Comment by jonrandy** [comment]
Well, as I said, I'm getting a failure on the 'not found' event test. It looks like the router code is trying to catch the error that occurs when trying to mount the tag type that is unknown to Riot, but the exception that is being thrown in the router code (in the timer) is stopping the test from running successfully (router dies?)

**Comment by jonrandy** [comment]
Removing or delaying the throwing of the exception seems to remove the problem, but I wasn't sure this was a sensible change to be making

**Comment by jonrandy** [comment]
Looks like I have more to do too, since Riot now has its own server side rendering...  I'll take a look at this soon

**Comment by jonrandy** [comment]
Also, is riot-ssr still needed? Riot seems to have its own server side rendering built in now

**Comment by jonrandy** [comment]
Made that update to do the async rendering just using riot's `renderAsync`. Does this successfully remove the need for riot-ssr? Or was riot-ssr used to get around having to do the `trigger('ready')` thing to resolve the promise returned by `riot.renderAsync`

**Comment by prateekbh** [comment]
@jonrandy riot-ssr was to do async kind rendering, if riot's renderAsync wis serving the purpose feel free to remove it.

I was stuck between build systems hence merged your request, but current build is failing.
Would love to work on a new PR

**Comment by prateekbh** [comment]
Hey @jonrandy!
I've fixed my ugly code 😞
Its now smooth and passing all tests, If you are going to do any more changes let me know. Else I'll go ahead and publish the new package.

**Comment by jon-paysbuy** [comment]
Sorry for the delayed reply - no more changes from me. Go ahead and publish

**Comment by jonrandy** [comment]
Sorry for the delayed reply - no more changes from me. Go ahead and publish

**Pull request merged**